onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -455,6 +459,11 @@ def _where_input_wrangler(
         matcher=lambda sample: sample.kwargs.get("as_tuple") is not None,
         reason="as_tuple=True is not supported",
     ),
+    skip(
```

Comment:
What are some other possible types? How can we account for them? Is this a case where the OpInfo does not match the aten op exactly? If that’s the case I would put in reason “non float values for the argument 'mean' are not expected by the aten op”

Reply:
the torch's test case also have Tensor type for 'mean', this is not supported by ort's op.RandomNormalLike().

Reply:
ORT only accept float type for args[0] 'mean'